### PR TITLE
Replace Compass with node-sass

### PR DESCRIPTION
**Pull request by MKodde**
These tasks have been applied:
1. Node SASS was installed
2. Grunt-sass was installed
3. Compass task removed from Gruntfile
4. Compass stuff was stripped from the EB sass files
5. Compass reset is missing now, resulting in messed up styling
6. Update readme

Todo:
1. ~~Remove compass and ruby from the project and OC-deploy~~ Not required as these have already been stripped. An update of the readme should be sufficient.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +27,9 @@ In short, themes require compilation which can be done by running the following
 To setup the required tooling on the VM, the following steps might be useful:
 
     cd /opt/openconext/OpenConext-engineblock/theme
-    sudo yum install gcc
```
Comment (pablothedude) [inline README.md]: Don't we need this perhaps for npm?

File: README.md [hunk 2]
```diff
@@ -27,14 +27,9 @@ In short, themes require compilation which can be done by running the following
...
     sudo curl --silent --location https://rpm.nodesource.com/setup_8.x | sudo bash -
     sudo yum install nodejs
     sudo npm install -g grunt grunt-cli
-    
-    sudo yum install gem ruby-devel
```
Comment (pablothedude) [inline README.md]: :100:

**Comment by pablothedude** [comment]
There was an error while installing all dependencies through npm. I've fixed that.

File: README.md (hunk 1 above)
Comment (MKodde) [reply README.md]: Judging from your comment below:

> There was an error while installing all dependencies through npm. I've fixed that.

We might need to add `sudo yum install gcc` back in the README.md?

**Pull request merged**